practice_flow_control_and_methods/grade_evaluator.rb

```diff
@@ -0,0 +1,24 @@
+# Ask for a grade (0–100). Output:
+
+# 90+ → A
+# 80–89 → B
+# 70–79 → C
+# 60–69 → D
+# < 60 → F
+
+puts "give me a grade (0-100)"
+given_grade = gets.chomp.to_f
+
+if given_grade >= 90.00
+  puts "You got an A!!"
+elsif given_grade <= 89.99 && given_grade >= 80
```

Comment:
instead of saying
`given_grade <= 89.99`
you should just say
`given_grade < 90`

remember in math?
```
 89     90     91
         *------------>    means  >= 90
---------O                 means  < 90
```